wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -31,7 +31,7 @@ fun stringForDomain(domain: String, context: Context): String? =
         )
         )[domain]?.let { context.getString(it) }
 
-fun getIcon(icon: String?, domain: String, context: Context): IIcon? {
+fun getIcon(icon: String?, domain: String, context: Context?): IIcon? {

```

Comment:
Why are you changing `Context` to `Context?` here? It should still always be available

Reply:
Oops! Something I forgot to remove when trying out different methods. Fixing.